chrome/locale/en-US/zotero/preferences.ftl

```diff
@@ -59,6 +59,9 @@ preferences-locate-library-lookup-intro = Library Lookup can find a resource onl
 preferences-locate-resolver = Resolver:
 preferences-locate-base-url = Base URL:
 
+preferences-tags-fuzzy-autocomplete =
+    .label = Fuzzy tags autocomplete
```

Comment:
(This wouldn't be "fuzzy" matching. We could actually do that, but that would imply finding strings that were similar but not exact. This is just a non-left-bound search.)